rpc/comms/ipc_unix.go

```diff
@@ -50,15 +51,16 @@ func (self *ipcClient) reconnect() error {
 func startIpc(cfg IpcConfig, codec codec.Codec, api shared.EthereumApi) error {
 	os.Remove(cfg.Endpoint) // in case it still exists from a previous run
 
-	l, err := net.ListenUnix("unix", &net.UnixAddr{Name: cfg.Endpoint, Net: "unix"})
+	l, err := net.Listen("unix", cfg.Endpoint)
```

Comment:
Yes. Using `ListenUnix` is sort-of silly anyway if the additional methods aren't used.